### ISSUE TITLE
poor performance

### ISSUE THREAD

**Issue by victorshengwgit** [Tensor-king/SA_Uet-pytorch#3]
hi bro，thanks for your repo!! I train the net with your guidance, but I comment the lines about wandb. However I found the loss and evaluation are not that good as yours. and the results visualization is like this...![0](https://github.com/Tensor-king/SA_Uet-pytorch/assets/87215766/8b1a32b8-2664-4a31-b3d0-bcd04865881d)
if I  make` output [output > 0.5] =255`,it will become whole black. I failed to debug and find out how to fix it. Could you give some advice?

[jobbb_SA_Net_train_CHASEDB1.66966 -.md](https://github.com/Tensor-king/SA_Uet-pytorch/files/12481610/jobbb_SA_Net_train_CHASEDB1.66966.-.md)

**Comment by victorshengwgit** [comment]
sorry it is my fault， I commented all code about wandb. but sth in wandb is necessary！
